glossary.yml

```diff
@@ -8739,7 +8778,10 @@
       A [repository](#repository) located on another computer. Tools such as [Git](#git) are
       designed to synchronize changes between local and remote
       repositories in order to share work.
-
+  uk:
+    term: "віддалений репозиторій"
+    def: >
+      [Репозиторій](#repository), розташований на іншому сервері або пристрої. 
```

Comment:
Shall we add the sync part to the "Git" definition instead?

Reply:
Not sure about changing the Git definition - but the English text here has two sentences, so why to ignore the 2nd?

Reply:
I'd use your suggested text here, to maintain closer 1-to-1 mapping between terms and their translations. Note also that Git novices would now have known about local and remotes from the start, so that may be a cognitive overload. We don't want the reader think "I am not collaborating on this project with others, so I don't need version control.

I'd just add the sentence here, "За допомогою системи контролю версій, такої як Git, можна синхронізувати локальні та віддалені репозиторії, що дозволяє спільну роботу."

Reply:
The rest now looks good, that's the only term left!

Reply:
@YehorBoiar this went to an older thread - pinging you here to help to see it

Reply:
> За допомогою системи контролю версій, такої як Git, можна синхронізувати локальні та віддалені репозиторії, що дозволяє спільну роботу.

Agree.